### PR TITLE
implement  add-hosts flag && extra_hosts 

### PR DESCRIPTION
**Pull request by fahedouch**
Hello

- implement `add-host` flag
- implement  compose `extra_hosts`

related ticket https://github.com/containerd/nerdctl/issues/314

### CODE REVIEW

File: cmd/nerdctl/internal_oci_hook.go [hunk 1]
```diff
@@ -26,6 +26,12 @@ var internalOCIHookCommand = &cli.Command{
 	Name:   "oci-hook",
 	Usage:  "OCI hook",
 	Action: internalOCIHookAction,
+	Flags: []cli.Flag{
+		&cli.StringSliceFlag{
+			Name:  "add-host",
+			Usage: "Add a custom host-to-IP mapping (host:ip)",
+		},
+	},
```
Comment (AkihiroSuda) [inline cmd/nerdctl/internal_oci_hook.go:34]: Why not use labels as in other flags?

Comment (fahedouch) [reply cmd/nerdctl/internal_oci_hook.go:34]: @AkihiroSuda you mean rename `add-host` to `labels` ?

Comment (AkihiroSuda) [reply cmd/nerdctl/internal_oci_hook.go:34]: No, using labels in https://github.com/containerd/nerdctl/blob/master/pkg/labels/labels.go

Other ocihook features use these labels

Comment (fahedouch) [reply cmd/nerdctl/internal_oci_hook.go:34]: @AkihiroSuda  fixed

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**